pyemby/emby.py

```diff
@@ -98,3 +98,22 @@ def get_image(self, item_id, style, played=0):
         """Return media image."""
         return '{0}/Items/{1}/Images/{2}?api_key={3}&PercentPlayed={4}'.format(
             self.server_url, item_id, style, self.api_key, played)
+
+    def get_latest_items(self, session, limit=3, is_played='false',
+                         include_item_types='episode'):
+        """Return XX most recent movie or episode additions to library"""
+        query = 'IncludeItemTypes={0}&Limit={1}&IsPlayed={2}'
+        query = query.format(include_item_types, limit, is_played)
+        try:
+            response = self.emby_request.get(self.server_url +
+                                             '/Users/{0}/Items/Latest?{1}'.
+                                             format(session['UserId'],
+                                                    query))
+        except KeyError as err:
+            _LOGGER.error('Requests error getting key: %s', err)
+            return
+        except requests.exceptions.RequestException as err:

```

Comment:
You can combine this with the KeyError so there is only one exception line:
```python
except (KeyError, requests.exceptions.RequestException) as err:
```
Then just make the message a little less specific.